quickstart.schema.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
@wtrocki Just making sure that these capitalization tweaks in this json file aren't a problem for building or anything, correct?